bindings/py/cpp_src/bindings/engine/py_Engine.cpp

```diff
@@ -52,6 +52,9 @@ using namespace nupic;
 
 namespace nupic_ext
 {
+
+	typedef std::shared_ptr<Region> Region_Ptr_t;
```

Comment:
nitpick: replace tabstops with 4 spaces.

Reply:
> nitpick: replace tabstops with 4 spaces.

yeah, we have not been very consistent on that.  Many of the modules use 2 space indent...or even 1 space.